### PR TITLE
CC2650: enable bootloader backdoor

### PR DESCRIPTION
**Pull request by librasungirl**

### CODE REVIEW

**Comment by jwhui** [comment]
Agree with @srickardti.  In general, we should keep `third_party` code unmodified.

**Comment by librasungirl** [comment]
Thanks, will update

**Comment by jwhui** [comment]
@srickardti, please review.

File: examples/platforms/cc2650/cc2650_ccfg.c [hunk 1]
```diff
@@ -30,6 +30,18 @@
  * Configure the Customer Configuration Area.
  */
 
+// enable bootloader backdoor
+#define SET_CCFG_BL_CONFIG_BOOTLOADER_ENABLE            0xC5       // Enable ROM boot loader
+
+#define SET_CCFG_BL_CONFIG_BL_LEVEL                     0x0        // Active low to open boot loader backdoor
+
+#define SET_CCFG_BL_CONFIG_BL_PIN_NUMBER                0x0D       // DIO13 (BTN-1 button) on CC2650 LauchPad Board for boot loader backdoor
```
Comment (srickardti) [inline examples/platforms/cc2650/cc2650_ccfg.c]: nit: spelling error in LaunchPad

Comment (librasungirl) [inline examples/platforms/cc2650/cc2650_ccfg.c]: thanks. updated

**Pull request merged**